### PR TITLE
PHP Migration examples

### PR DESCRIPTION
**Pull request by barbara79**

### CODE REVIEW

File: Migration.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
+# Guide to migrate from the PHP SDK 1.x to the PHP SDK 2.x
+
+This guide helps developers to migrate from the version 1.x to the version 2.x of PHP SDK.
+
+## Table of content
+
+* [Client Configuration and Creation](#client-configuration-and-creation)
+* [Timeout Setting](#timout-setting)
```
Comment (jherey) [inline Migration.md:8]:
```suggestion
* [Timeout Setting](#timeout-setting)
```

File: Migration.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+* [Headers](#headers)
+* [Retry](#retry)
+* [Draft Builder](#draft-builder)
+* [Create Command](#create-command)
+* [Create From Json](#create-from-json)
+* [Update Command](#update-command)
+* [Query - GetById](#query-getbyid)
```
Comment (jherey) [inline Migration.md:15]:
```suggestion
* [Query - GetById](#query-get-by-id)
```

File: Migration.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+        $clientSecret = $_ENV['CTP_CLIENT_SECRET'] ?? '';
+        $projectKey = $_ENV['CTP_PROJECT'] ?? '';
+        $authConfig = new ClientCredentialsConfig(new ClientCredentials($this->clientId, $this->clientSecret), null, self::OAUTH_URL);
+        $client = ClientFactory::of()->createGuzzleClient(new Config(['maxRetries' => 3], self::API_URL), $authConfig);
+        $apiRequestBuilder = new ApiRequestBuilder($client);
+        $request = $apiRequestBuilder->withProjectKey($this->projectKey)->get();
+        $client = $request->execute();
+```
+
+<a id="timout-setting"></a>
```
Comment (jherey) [inline Migration.md:54]:
```suggestion
<a id="timeout-setting"></a>
```

File: Migration.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+        /** @var Category $category */
+        $request = $builder->with()->categories()->withId($category->getId())
+            ->post(
+                CategoryUpdateBuilder::of()->withVersion($category->getVersion())
+                    ->withActions($updateCollection)->build()
+            );
+        $categoryUpdated = $request->execute();
+```
+
+<a id="query-getbyid"></a>
```
Comment (jherey) [inline Migration.md:266]:
```suggestion
<a id="query-get-by-id"></a>
```

**Review by jherey — COMMENTED** [review]
Good work @barbara79. Just some nitpicks

File: Migration.md (hunk 4 above)
Comment (barbara79) [reply Migration.md:253]: I'm also changing directly the header with Get By Id  👍

**Review by jherey — APPROVED** [review]

**Pull request merged**